engine/api/index.md

```diff
@@ -119,10 +117,9 @@ You can specify the API version to use, in one of the following ways:
 - The Docker Go SDK allows you to enable API version negotiation, automatically
   selects an API version that is supported by both the client, and the Docker Engine
   that is used.
-
 - For the SDKs, you can also specify the API version programmatically, as a
   parameter to the `client` object. See the
-  [Go constructor](https://github.com/moby/moby/blob/v19.03.6/client/client.go#L119){: target="_blank" rel="noopener" class="_"}
+  [Go constructor](https://pkg.go.dev/github.com/docker/docker@v23.0.5+incompatible/client#NewClientWithOpts){: target="_blank" rel="noopener" class="_"}
```

Comment:
Maybe we shouldn't pin the version here, it's gonna go out of date quickly

Instead we could point to:

https://pkg.go.dev/github.com/docker/docker/client#NewClientWithOpts

Reply:
Ah, perhaps we can now, yes. Previously Go modules would pick `v1.13.1` as "latest", which may have been why we used a fixed version.